lib/components/Selection control/Checkbox/index.tsx

```diff
@@ -0,0 +1,117 @@
+import {
+    ComponentProps,
+    forwardRef,
+    ReactNode,
+    useState,
+    useEffect,
+} from "react";
+import clsx from "clsx";
+import "./checkbox.scss";
+import {
+    StandaloneSquareCheckFillIcon,
+    StandaloneSquareBoldIcon,
+} from "@deriv/quill-icons";
+import { Text } from "../../Typography";
+
+interface CheckboxProps
+    extends Omit<ComponentProps<"input">, "placeholder" | "defaultChecked"> {
+    fontSize?: "sm" | "md" | "lg" | "xl";
```

Comment:
```suggestion
    size?: "sm" | "md" | "lg" | "xl";
```

Can we minimize it to `size`?

Reply:
Sure, will do